src/test/java/com/karankumar/bookproject/backend/statistics/PageStatisticsTest.java

```diff
@@ -89,6 +89,14 @@ void averagePageLengthExistsAndIsCorrect() {
         assertEquals(averagePageLength, pageStatistics.calculateAveragePageLength());
     }
 
+    @Test
+    void averagePageLengthWhenRoundedToHigherNumber() {
+        StatisticTestUtils.deleteBook(StatisticTestUtils.getBookWithHighestRating());
+        pageStatistics = new PageStatistics(predefinedShelfService);
+        int averagePageLength = (int) Math.ceil(StatisticTestUtils.getTotalNumberOfPages() / (float) StatisticTestUtils.getNumberOfBooks());
```

Comment:
Could you please ensure this line adheres to our 100 character column limit?

Reply:
Also, could you please replace this line with a hardcoded value? I think the problem with the current approach is that we're redefining the code under test, so this should always pass.